src/service/token.service.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-useless-return */
+import { Token } from '../moduls/Token.js';
```

Comment:
There is a typo in the import path: 'moduls' should be 'modules'. Please correct the path to '../modules/Token.js'.